app/views/admin/events/show.html.erb

```diff
@@ -31,7 +31,9 @@
         </div>
         <div class="row">
           <div class="col-lg-2 col-md-12 field-key">Description</div>
-          <div class="col-lg-10 col-md-12 field-value"><%= @event.event_info.description %></div>
+          <div class="col-lg-10 col-md-12 field-value">
+            <%= raw sanitize @event.event_info.description, tags: %w(strong em a br), attributes: %w(href) %>
```

Comment:
Here you only sanitize the bold, italic, newline, and link. Meanwhile in the text editor, there is actually no support for link (a href), and here you also don't sanitize the underline effect... probably can consider to update this part.

Also, when I tried the text editor, the box height kept increasing as I typed new line or add more text, can it be adjusted to just show a fixed height and scroll text if the box if not enough to contain all the content.

Reply:
Noted!